devspaces-udi/etc/entrypoint.sh

```diff
@@ -70,4 +70,50 @@ if [[ ! -z "${PLUGIN_REMOTE_ENDPOINT_EXECUTABLE}" ]]; then
   ${PLUGIN_REMOTE_ENDPOINT_EXECUTABLE}
 fi
 
+#############################################################################
+# If KUBEDOCK_ENABLED="true" then link podman to /usr/bin/podman.wrapper
+# else link podman to /usr/bin/podman.orig
+#############################################################################
+if [ "${KUBEDOCK_ENABLED:-false}" = "true" ]; then
+    echo
+    echo "Kubedock is enabled (env variable KUBEDOCK_ENABLED is set to true)."
+
+    SECONDS=0
+    until [ -f /home/user/.kube/config ]; do
+        if (( SECONDS > 10 )); then
+            echo "Giving up..."
+            exit 1
+        fi
+        echo "Kubeconfig doesn't exist yet. Waiting..."
+        sleep 1
+    done
+    echo "Kubeconfig found."
+
+    KUBEDOCK_PARAMS=${KUBEDOCK_PARAMS:-"--reverse-proxy"}
+
+    echo "Starting kubedock with params \"${KUBEDOCK_PARAMS}\"..."
+    
+    kubedock server "${KUBEDOCK_PARAMS}" > /tmp/kubedock.log 2>&1 &
+    
+    echo "Done."
+
+    echo "Replacing podman with podman-wrapper..."
```

Comment:
can we be consistent with the filename all the way through? Why rename `-wrapper.sh` to `.wrapper` ? 

```suggestion
    echo "Replacing podman with podman-wrapper.sh..."
```